src/test/scala/com/stratio/hermes/kafka/KafkaProducerTest.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.hermes.kafka
+
+import java.util
+import java.util.Properties
+
+import com.typesafe.config.ConfigFactory
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.io.Source
+import scala.util.Try
+
+
+@RunWith(classOf[JUnitRunner])
+class KafkaProducerTest extends FlatSpec with Matchers {
+
+  val poll_time = 1000
```

Comment:
If it is a Scala constant I think that is better to follow the Scala Style: PollTime = 1000 instead of poll_time